lib/validators/options_processor/options/lib/andOrProcessor.js

```diff
@@ -24,8 +24,15 @@ const andOrProcessor = (validatorContext, input, rule, key) => {
   let isMergeable
   let DEFAULT_SUCCESS_MESSAGE = `All the requisite validations passed for '${key}' option`
   let descriptions = ''
+  let errorExists = false
+  let errorMessage = 'Error occurred: '
 
   validated.forEach(result => {
+    if (result.status === 'error') {
+      errorExists = true
+      errorMessage += result.description
```

Comment:
Do you have a test case for multiple errors from different options and is it possible to use `MD` bullet points or plain text carriage return formatting?

Reply:
![image](https://user-images.githubusercontent.com/5243508/77223198-96676600-6b17-11ea-8f56-d0f995ea95e7.png)